src/extension.ts

```diff
@@ -158,14 +159,26 @@ function diagnosticCounter(lc: LanguageClient) {
 }
 
 function registerCommands(lc: LanguageClient, context: ExtensionContext) {
-    const cmdDisposable = commands.registerTextEditorCommand('rls.deglob', (textEditor, _edit) => {
+    let cmdDisposable = commands.registerTextEditorCommand('rls.deglob', (textEditor, _edit) => {
         lc.sendRequest('rustWorkspace/deglob', { uri: textEditor.document.uri.toString(), range: textEditor.selection })
             .then((_result) => {},
                   (reason) => {
                 window.showWarningMessage('deglob command failed: ' + reason);
             });
     });
     context.subscriptions.push(cmdDisposable);
+
+    cmdDisposable = commands.registerTextEditorCommand('rls.findImpls', (textEditor: TextEditor, _edit: TextEditorEdit) => {
```

Comment:
To clarify, I think this is OK now, but could be fragile. For example, if we introduce a closure here we might end up pushing the second object twice and skipping the first (I admit this is unlikely, but the alternative is almost as easy).